### ISSUE TITLE
Persist httpHeaders on refresh

### ISSUE THREAD

**Issue by pantharshit00** [prisma/graphql-playground#909]
#### This issue pertains to the following package(s):

- [X] GraphQL Playground HTML
- [X] GraphQL Playground

### Context
https://github.com/prisma/prisma/issues/3707

#### What OS and OS version are you experiencing the issue(s) on?
MacOS Mojave 10.14.1

#### What version of graphql-playground(-electron/-middleware) are you experiencing the issue(s) on?
1.8.5

#### What is the expected behavior?
The playground should persist the httpHeaders on refresh. Many users are getting frustrated by this behavior of not persisting the headers even though we persist the query.

#### What is the actual behavior?
Playground simply erases the httpHeaders on refresh.

#### What steps may we take to reproduce the behavior?
1. Goto Playground.
2. Add an HTTP header.
3. Refresh the page. Boom the header is gone and we need to regenerate the token now.

**Comment by Huvik** [comment]
You need to set in settings: `"editor.reuseHeaders": true,` and then it should work.

**Comment by ebicoglu** [comment]
Hi,

I have the same issue. Only the `cookie` header is not being sent but the other headers are being sent.

![image](https://user-images.githubusercontent.com/9526587/54121143-bba49d00-440a-11e9-8189-4b6581fa30c6.png)

---

I set in settings: "editor.reuseHeaders": true,
![image](https://user-images.githubusercontent.com/9526587/54121206-e8f14b00-440a-11e9-8803-53454fd6738d.png)

---

I've installed `graphql-playground-electron-setup-1.8.10.exe`

![image](https://user-images.githubusercontent.com/9526587/54121308-2f46aa00-440b-11e9-952c-73aa0dc646a7.png)

---

By the way I tried to see my electron app version but the menu items `About GraphQL Playground` and `Check for Updates` not working in Windows 10.
![image](https://user-images.githubusercontent.com/9526587/54121390-5ac99480-440b-11e9-8c29-092575994390.png)
